packages/api/prisma/schema.prisma

```diff
@@ -17,13 +17,46 @@ datasource db {
 /// A user who can authenticate into the system.
 model User {
   /// The ID of the user.
-  id            String    @id @default(dbgenerated("generate_ulid()")) @db.Uuid
+  id            String           @id @default(dbgenerated("generate_ulid()")) @db.Uuid
   /// The display name of the user.
   name          String?
   /// The email address for logging in and notifications.
-  email         String?   @unique
+  email         String?          @unique
   /// The time that the email was last verified.
   emailVerified DateTime?
+  /// The list of system roles this user has
+  systemRoles   UserSystemRole[]
+  /// The list of sessions for the user.
+  sessions      Session[]
+}
+
+/// A authenticated session for a user.
+model Session {
+  id           String   @id @default(dbgenerated("generate_ulid()")) @db.Uuid
+  /// The session token from the cookie.
+  sessionToken String   @unique
```

Comment:
I ended up moving back to server sessions instead of JWTs so that if roles change, the JWT isn't out of date